tools/springboard-alerts/collect-all-alerts.rb

```diff
@@ -18,10 +18,22 @@ def add_entries_to_collection(lang, file_path, dict, collection)
 
 def collect_localization_dictionary(lang_dir_path)
   dict = {}
-  Dir.glob("#{lang_dir_path}/*.strings") do |file_path|
-    pairs = read_strings(file_path)
-    dict.merge!(pairs)
+  
+  3.times do |counter|
```

Comment:
You don't use counter variable inside loop, so it could be removed:
3.times do ... end